src/monsters.h

```diff
@@ -193,7 +196,7 @@ class Monsters
 		void loadLootContainer(const pugi::xml_node& node, LootBlock&);
 		bool loadLootItem(const pugi::xml_node& node, LootBlock&);
 
-		std::map<std::string, MonsterType> monsters;
+		std::unordered_map<std::string, MonsterType> monsters;
```

Comment:
Is there a particular reason you changed map to unordered map? Can you make sure that the resulting iterator instability on insert doesn't cause silent bugs?